js/ui/diagram/ui.diagram.toolbar.js

```diff
@@ -90,7 +93,7 @@ class DiagramToolbar extends Widget {
             return;
         } else if("value" in widget.option()) {
             widget.option("value", value);
-        } else {
+        } else if(value !== undefined) {
```

Comment:
null? isDefined?

Reply:
the diagram core returns non-undefined value only for commands which have a value. Other commands' value is undefined.